### ISSUE TITLE
failed at pve-manager/6.3-4/0a38c56f (running kernel: 5.4.98-1-pve)

### ISSUE THREAD

**Issue by Axxxolotl** [TheGrandWazoo/freenas-proxmox#93]
From deb nothing happend, in UI absent freenas iscsi provider.
then I tried on clean proxmox manuality patching and it answere:

root@sol-pve1:\~/freenas-proxmox# patch -b /usr/share/pve-manager/js/pvemanagerlib.js < pve-manager/js/pvemanagerlib.js.patch
patching file /usr/share/pve-manager/js/pvemanagerlib.js
Hunk #1 FAILED at 6183.
Hunk #2 FAILED at 32992.
Hunk #3 FAILED at 33004.
Hunk #4 succeeded at 46736 (offset 13711 lines).
Hunk #5 succeeded at 46753 with fuzz 1 (offset 13711 lines).
Hunk #6 FAILED at 33052.
Hunk #7 FAILED at 33067.
Hunk #8 succeeded at 46810 with fuzz 2 (offset 13711 lines).
Hunk #9 FAILED at 33109.
6 out of 9 hunks FAILED -- saving rejects to file /usr/share/pve-manager/js/pvemanagerlib.js.rej
root@sol-pve1:\~/freenas-proxmox# patch -b /usr/share/perl5/PVE/Storage/ZFSPlugin.pm < perl5/PVE/Storage/ZFSPlugin.pm.patch
patching file /usr/share/perl5/PVE/Storage/ZFSPlugin.pm
Hunk #5 succeeded at 172 with fuzz 1 (offset 4 lines).
Hunk #6 succeeded at 199 (offset 4 lines).
Hunk #7 succeeded at 244 (offset 4 lines).
Hunk #8 succeeded at 280 (offset 4 lines).
root@sol-pve1:\~/freenas-proxmox# patch -b /usr/share/pve-docs/api-viewer/apidoc.js < pve-docs/api-viewer/apidoc.js.patch
patching file /usr/share/pve-docs/api-viewer/apidoc.js
Hunk #1 succeeded at 39527 (offset 4526 lines).
Hunk #2 succeeded at 39731 (offset 4549 lines).
Hunk #3 succeeded at 40002 (offset 4609 lines).

**Comment by yysung1123** [comment]
Hi,
It seems that there's commas appended at the end of lines in the /usr/share/pve-manager/js/pvemanagerlib.js (pve-manager 6.3-4)

For instance,
```diff
Ext.define('PVE.form.iScsiProviderSelector', {
    extend: 'Proxmox.form.KVComboBox',
    alias: ['widget.pveiScsiProviderSelector'],
    comboItems: [
	['comstar', 'Comstar'],
	['istgt', 'istgt'],
	['iet', 'IET'],
-	['LIO', 'LIO']
+	['LIO', 'LIO'],
```

So the current patch file may fail to patch the new pvemanagerlib.js.

There's a modified patch file for pve-manager 6.3-4.

https://gist.github.com/yysung1123/87fe48dfa112835b187ddc4595defdb6

**Comment by TheGrandWazoo** [comment]
Updating patch to reflect changes in the pvemanagerlib.js so the patch get applied correctly. Seems that they added commas to some of the lines.
Looks to only effect the GUI.

**Comment by TheGrandWazoo** [comment]
Issue is closed.
Fix in package 2.0.2-3.

**Comment by hunter-nl** [comment]
> Issue is closed.
> Fix in package 2.0.2-3.

Is this fix also included into the deb package 2.0.2-3-beta1? As this one I just got upgraded to and do not see the FreeNas option in the iSCSI provider list anymore.
I checked both stable and beta repository.
